manifests/s/SillyKoala/IdleEucalyptus/1.1.0/SillyKoala.IdleEucalyptus.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SillyKoala.IdleEucalyptus
+PackageVersion: 1.1.0

```

```suggestion
PackageVersion: 1.1
```
This must match Control Panel.